rabbitmq/configure.sls

```diff
@@ -0,0 +1,128 @@
+#!pyobjects
```

Comment:
Was there a particular reason this needed to be a pyobjects instead of the default jinja-yaml pipeline? my thoughts are that it better to write `determine_(ram|disk)_limit`.